packages/components/src/components/header/header.component.ts

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable @typescript-eslint/unbound-method */
+import { classMap } from 'lit/directives/class-map.js';
+import { html } from 'lit/static-html.js';
+import { property } from 'lit/decorators.js';
+import type { CSSResultGroup } from 'lit';
+import SynergyElement from '../../internal/synergy-element.js';
+import { HasSlotController } from '../../internal/slot.js';
+import componentStyles from '../../styles/component.styles.js';
+import styles from './header.styles.js';
+
+/**
+ * @summary The <syn-header /> element provides a generic application header
+ * that can be used to add applications name, toolbar and primary navigation.
+ *
+ * @documentation https://synergy-design-system.github.io/?path=/docs/components-syn-header--docs
+ * @status stable
+ * @since 1.8.0
+ *
+ *
+ * @slot - The label for the header. Will automatically be hidden on mobile.
+ * @slot logo - The logo that should be displayed. Will fall back to the SICK logo if not applied.
+ * @slot option-menu - Used to add various application toolbar icons.
+ *                     Best used with `<syn-icon-button />` and `<syn-drop-down />`
+ * @slot top-navigation - Used to add an optional horizontal navigation
+ *
+ * @csspart base - The component's base wrapper.
+ * @csspart side-navigation-button - Can be used to apply styles to the side-navigation menu icon
+ * @csspart logo - The wrapper where the application logo resides in
+ * @csspart label - Wrapper of the application name label
+ * @csspart option-menu - Item that wraps the optional application menu
```

Comment:
I know that this comes from design – but reading this, I'm not really happy with the wording. We have an `syn-option` and maybe a `syn-menu` and `syn-menu-item` (see https://shoelace.style/components/menu).

Both have nothing to do with this area. It's not a menu.

What do you think about calling it `toolbar`? Or `tools` or similar? Would love to read some opinions on that @kirchsuSICKAG @reichst1SICKAG @LeaLeRose

Reply:
On the website we have those icons also on the top-right and we are calling it `meta-navigation`

`toolbar` could be also anywhere in the main application.

Reply:
`meta-navigation` is nice, especially as this could contain some kind of nav items as well (e. g. to switch apps). What does the others think?

Reply:
Think so, too. Lets stick with `meta-navigation`.

Reply:
I opted to use `meta-navigation` as the slots title. Will be available on next push.